packages/apps-config/src/endpoints/testingRelayWestend.ts

```diff
@@ -64,6 +64,24 @@ export function createWestend (t: TFunction): EndpointOption {
           Centrifuge: 'wss://fullnode-collator.charcoal.centrifuge.io'
         }
       },
+      {
+        info: 'darwinia',
+        homepage: 'https://darwinia.network/',
+        paraId: 2003,
+        text: t('rpc.darwinia.network', 'Darwinia', { ns: 'apps-config' }),
+        providers: {
+          'Standard Protocol': 'wss://rpc.darwinia.network'
+        }
+      },
+      {
+        info: 'darwinia',
+        homepage: 'https://darwinia.network/',
+        paraId: 2102,
+        text: t('rpc.darwinia.network', 'Darwinia', { ns: 'apps-config' }),
+        providers: {
+          'Standard Protocol': 'wss://rpc.darwinia.network'
```

Comment:
Not sure how 2 paradise can point to the same endpoint - a runtime can only have a single paraId configured?